src/cf-pagination/src/molecules/pagination.less

```diff
@@ -0,0 +1,107 @@
+//
+// Theme variables
+//
+
+// Color variables
+// $color- variables are from 18F's US Web Design Standards
+// https://github.com/18F/web-design-standards/blob/18f-pages-staging/src/stylesheets/core/_variables.scss
+
+@pagination-text-color: #757575; // $color-gray-medium
+@pagination-bg-color: #f1f1f1; // $color-gray-lightest
+
+// Sizing variables
+
+@pagination-font-size-px: @base-font-size-px;
+@pagination-font-size-em: unit( @pagination-font-size-px / @base-font-size-px, em );
+@pagination-btn-height-px: 42px;
+@pagination-btn-width-px: 130px;
+
+//
+// Pagination
+//
+
+.m-pagination {
+    position: relative;
+
+    &_form {
+        padding: unit( 5px / @base-font-size-px, em );
+
+        border-radius: unit( 4px / @base-font-size-px, em );
+        background: @pagination-bg-color;
+        color: @pagination-text-color;
+        text-align: center;
+    }
+
+    // Currently requires explicit input[type="number"] due to cascade issue
+    // TODO: Fix cascade issue, possibly due to import order
+    input[type="number"],
+    &_current-page {
+        .webfont-medium();
```

Comment:
Should you go ahead and update cf-core in this branch? Because when it does get updated, this will have to be set with `.u-webfont-medium();`.